src/main/java/com/twilio/http/Request.java

```diff
@@ -86,23 +92,23 @@ public void setAuth(final String username, final String password) {
         this.password = password;
     }
 
+    public void setRegion(final String region) {
+        this.region = region;
+    }
+
+    public void setEdge(final String edge) {
+        this.edge = edge;
+    }
+
     /**
      * Create auth string from username and password.
      *
      * @return basic authentication string
      */
     public String getAuthString() {
-        String credentials = this.username + ":" + this.password;
-        try {
-
-            String encoded = DatatypeConverter.printBase64Binary(credentials.getBytes("ascii"));
-            return "Basic " + encoded;
-
-        } catch (final UnsupportedEncodingException e) {
-
-            throw new InvalidRequestException("It must be possible to encode credentials as ascii", credentials, e);
-
-        }
+        final String credentials = this.username + ":" + this.password;
+        final String encoded = DatatypeConverter.printBase64Binary(credentials.getBytes(StandardCharsets.US_ASCII));
+        return "Basic " + encoded;
```

Comment:
Why are we removing the try/catch? Concerned we are breaking someone's implementation expecting this exception, though that seems unlikely and/or rare.

Reply:
It's no longer possible the throw `UnsupportedEncodingException` here because we're using `StandardCharsets.US_ASCII` instead of `"ascii"`.

Reply:
Should we mark this as a breaking change?

Reply:
It's not breaking. The exception should never have been thrown to begin with. It's meant to catch cases where the charset was not know until runtime.